src/verification_key.rs

```diff
@@ -8,10 +8,7 @@ use curve25519_dalek::{
 use sha2::Sha512;
 use zeroize::DefaultIsZeroes;
 
-pub use ed25519::{
-    signature::{Signer, Verifier},
-    Signature,
-};
+use ed25519::{signature::Verifier, Signature};
```

Comment:
The `verification_key` module is not `pub` so it's harmless to remove `pub` here